docs/source/model_design/vivarium_model_components/risk_factors/risk_correlation/index.rst

```diff
@@ -1,1213 +1,152 @@
-
-.. math::
-	PAF_{joint} = 1 - \frac{1}{\frac{1}{n}\sum_{i=1}^{n} RR_1^{e1_i} \cdot RR_2^{e2_i}}
-
-These two approaches give nearly the same value for the attributable fraction when the exposure to each risk is independent. However, when the exposures to the risk factors are correlated, the so-called Joint approach gives a higher value of the attributable fraction than the Multiplicative (current Vivarium) approach.
-We have conducted some simple tests to determine how this impacts the final rate of events among individuals. In this test, we calculate a risk deleted rate as:
-
-.. math::
-	i_0 = \left(1-\text{PAF}\right) \cdot i_{\text{GBD}}
-
-And calculate the attributable fraction in the two ways described above, as a multiplicative and a joint PAF that we compared.
+.. todo::
+  Add a more detailed description of when to include risk correlation, including
+  which results are biased by the omission of a risk correlation and
+  how to assess size/acceptability of the bias.
 
-.. math::
-	i_{multi,0} = (1-{PAF_{multi}}) \cdot i_{{GBD}}
-.. math::
-	i_{joint,0} = (1-{PAF_{joint}}) \cdot i_{{GBD}}
-
-
-Each individual then has a rate of event defined by
-
-.. math::
-	i_{e_1, e_2} = i_0 \cdot \left(\text{RR}_1\right)^{e_1}\cdot \left(\text{RR}_2\right)^{e_2}
-
-Where i is the rate for an individual, i0 is the risk deleted rate, RR is a relative risk and e is an indicator {0,1} denoting if the individual is exposed to that risk factor.
-
-Our simple tests show that when evaluating these attributable fractions on individuals with a known rate of event, the joint PAFs return rates nearly identical to the known rate of events. In contrast, the multiplicative PAFs return rates that are too high when the risk exposures are correlated (rate here is 0.5).
-
-.. image:: rate_dotplot.jpg
-
-Proposal to test
------------------------
-We would like to test these two changes for exposures and attributable fractions in the Balanced Energy Protein model as sensitivity analyses supplementing the primary analysis. The risks that we want to test are childhood growth failure: stunting, underweight, and wasting. We are proposing using correlation structures for these exposures from analyses of individual-level data.
-
-Also in the BEP model, we would like to attempt creating correlated changes to propensity scores for stunting, underweight, and wasting within individuals over time. This would involve implementing multiple samples for these risk factors to determine propensity and risk exposure for each simulant. There would not be any changes to the attributable fractions for these risks, the only change would be in allowing these propensity values to change at defined time points, corresponding with GBD age groups.
-
-Illustrative code example
+Risk exposure correlation
```

Comment:
I think we could add a todo note here to include some info about how this can be done for non-continuous risks as well. Some approaches we've taken include:
- Combining two dichotomous risks into one risk with four categories of the joint distribution (which could scale beyond dichotomous)
- Assigning one risk exposure first and using that to inform conditional probability of second risk exposure assignment

Reply:
Here's an example of the latter: https://vivarium-research.readthedocs.io/en/latest/models/risk_exposures/maternal_bmi_hgb/index.html

Reply:
I've left out your first bullet point for now, because the rest of this page wouldn't really apply in that case (from a Vivarium perspective, it would be a single risk).